organizers/views.py

```diff
@@ -44,6 +44,10 @@ def add_comment(application, user, text):
     return comment
 
 
+def set_dubious(application):
+    application.set_dubious()
```

Comment:
Is this function used?

Reply:
I think this hasn't written by me, I call application object directly